app/konnect/org-management/teams-and-roles/manage.md

```diff
@@ -31,51 +33,47 @@ configuration page.
 You can edit the roles for any custom team. Any changes made to a team's roles
 are automatically applied to all team members.
 
-1. From the left navigation menu in {{site.konnect_short_name}}, open the {% konnect_icon organizations %}
- **Organization** link.
-2. Choose a team from the list.
+1. Open {% konnect_icon organizations %} [**Organization**](https://cloud.konghq.com/organization/) to go to the **Teams** page, then choose a team from the list.
 
     Teams with the `Predefined` label can't be edited or deleted.
 
-3. Choose an entity tab, then click **+ Add role(s)**.
+1. Open either the **Services** or **Runtime Groups** tab, then click **Add role(s)**.
```

Comment:
Hmm. I don't think either of those quite works, since it places "services" as more important.

Maybe this is an instance when I should actually split the step, ie:

> 1. Choose one of the entity tabs: Services or Runtime Groups.
> 2. Click **Add role**.

Going with something like that for now.